ios/FluentUI/Command Bar/CommandBarButton.swift

```diff
@@ -55,12 +55,12 @@ class CommandBarButton: UIButton {
         isEnabled = item.isEnabled
         isSelected = isPersistSelection && item.isSelected
 
-        if item.iconImage == nil {
-            setTitle(item.title, for: .normal)
-            if let font = item.titleFont {
-                titleLabel?.font = font
-            }
-        }
+        // always update icon and title as we only display one; we may alterenate between them, and the icon may also change
+        let iconImage: UIImage? = item.iconImage
```

Comment:
I think you don't need the explicit type here since item.iconImage is of type **UIImage?** .
You can just do the following:

`let iconImage = item.iconImage`

Reply:
Yep, I wanted the code to be clear that it was optional and we should also deliberately set to null, but the comment will cover that, thanks!